deployment/aws/package-lock.json

```diff
@@ -9,13 +9,13 @@
       "version": "0.1.0",
       "license": "MIT",
       "dependencies": {
-        "cdk": "1.160.0"
+        "cdk": "^1.165.0"
```

Comment:
Just one thing here, is it supposted to have the ^?

Reply:
yes thats a [special flag](y-theres-a-caret-in-my-package-json/) widening the scope of the version for backward compatibility. 
The whole idea however behind a package-lock file is that you will get a consistent installs no matter what environment you are installing it. That means the installed versions must be fixed to a specific version. and ^ is a bad habit. But its snyk who auto created the PR. So will accept and we can fix after.